vars/pipelineDeploy.groovy

```diff
@@ -180,6 +207,55 @@ void call() {
                     }
                 }
             }
+            stage('Backup') {
+                when {
+                    expression {
+                        env.NET_NAME == 'testnet'
+                    }
+                }
+                steps {
+                    script {
+                        veganet('chainstorecopy vegalogcopy')
+                    }
+                }
+            }
+            stage('Store checkpoint') {
+                when {
+                    expression {
+                        env.NET_NAME == 'testnet'
+                    }
+                }
+                steps {
+                    withCredentials([sshCredentials]) {
+                        script {
+                            netSsh = { command ->
+                                return 'ssh -t -i $PSSH_KEYFILE $PSSH_USER@n04.$NET_NAME.vega.xyz "sudo' + command + '"'
+                            }
+                            newestFile = sh (
+                                script: netSsh 'ls -t /home/vega/.local/state/vega/node/checkpoints/ | head -n 1',
+                                returnStdout: true
+                            ).trim()
+                            version = sh (
+                                script: netSsh "/home/vega/current/vega version | awk '{print $3}'"
+                            )
+                            sh netSsh "cp  /home/vega/.local/state/vega/node/checkpoints/${newestFile} /tmp/${newestFile}; chown `whoami`:`whoami` /tmp/${newestFile}"
+                            sh 'scp -i $PSSH_KEYFILE $PSSH_USER@n04.$NET_NAME.vega.xyz:/tmp/${newestFile} .'
+                            sh "mkdir -p checkpoint-store/Testnet-1/${version}"
+                            sh "mv ${newestFile} checkpoint-store/Testnet-1/${version}/"
```

Comment:
We are using the Fairground directory: https://github.com/vegaprotocol/checkpoint-store/tree/main/Fairground